<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundOptionsDialog</class>
 <widget class="QDialog" name="SoundOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Label" name="cTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Label" name="cStateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>208</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Audio State</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Label" name="cVolumeLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Button" name="cMusic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>208</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
  </widget>
  <widget class="Button" name="cSpeech">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>208</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Speech</string>
   </property>
  </widget>
  <widget class="Button" name="cSounds">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>208</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sound Effects</string>
   </property>
  </widget>
  <widget class="Button" name="cCity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>208</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>City Sounds</string>
   </property>
  </widget>
  <widget class="Button" name="cOk">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>192</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="Button" name="cCancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>192</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="Button" name="cMusicDown">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cMusicUp">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>80</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cSpeechDown">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cSpeechUp">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>110</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cSoundsDown">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cSoundsUp">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>140</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cCityDown">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label" name="cMusicVolumeLabel">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>78</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Label" name="cSpeechVolumeLabel">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>108</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Label" name="cSoundsVolumeLabel">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>140</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Label" name="cCityVolumeLevel">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>170</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Button" name="cCityUp">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>170</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>widget/button.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>widget/label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
